cli/src/core/config/viper.go

```diff
@@ -0,0 +1,6 @@
+package config
+
```

Comment:
The purpose of this file seems to be to act as a global store of data, I am usually against doing something like this unless it significantly reduces code effort. In acceptance of this I would encourage isolating this global storage use case into its own package and limit mutation as much as possible, additionally make it clear that this is being used as a state store maybe by renaming the package "store" or "state"

Reply:
It is being used as a config store for env files. It has to be this way because the envFiles is referenced by the persistent flags for the rootcmd, and the env files must be swapped out by the command line env files in place of profile env files if the user specifies a profile and env files.

The reason they exist in this package is to have a central place to retrieve global variables while avoiding circular dependencies.

But, agreed on the naming. I changed the package name to 'state' in [0f23ab8](https://github.com/openhie/package-starter-kit/pull/32/commits/0f23ab8389c5b0fb36b336be5f0c77cfcbd36707)